src/impl/websocketserver.cpp

```diff
@@ -40,9 +40,13 @@ WebSocketServer::WebSocketServer(Configuration config_)
 			    "Either none or both certificate and key PEM files must be specified");
 		}
 	}
-
+	
+	const char* bindAddress = nullptr;
+	if(config.bindAddress){
+		bindAddress = config.bindAddress->data();
```

Comment:
While this is valid since C++11 as `data()` now guarantees that a null character terminates the character sequence, I'd prefer a call to `c_str()` when getting a C-style string for clarity.